docs/kratos/emails-sms/05_custom-email-templates.mdx

```diff
@@ -0,0 +1,285 @@
+---
+id: custom-email-templates
+title: Customize message templates for recovery and verification emails
+sidebar_label: Email templates
+---
+
+Ory Identity Service (Ory Kratos) comes with built-in templates for all messages sent by the system. You can replace the default
+templates with custom ones that will carry your own branding, visual elements, and communication tone.
+
+The system uses templates in these flows:
+
+1. `recovery` - the system sends these messages when users recover their accounts, for example after forgetting their password.
+   - `recovery.valid` - the system sends these messages when the user completes the recovery flow successfully.
+   - `recovery.invalid` - the system sends these messages when the recovery flow fails.
+2. `verification` - the system sends these messages when users verify their accounts after signing up.
+   - `verification.valid` - the system sends these messages when the user verifies their account successfully.
+   - `verification.invalid` - the system sends these messages when the account verification fails.
```

Comment:
```suggestion
   - `verification.invalid` - the system sends these messages when the account verification fails for example when the account does not exist.
```
```suggestion
   - `verification.invalid` - the system sends these messages when the user tries to verify an account that does not exist.
```

Maybe we can make it clearer when these messages are sent?

Reply:
Sure! Did that for `recovery.invalid`, but your example doesn't really sound right... Are you sure you can try to verify an email that's not "registered"? I don't think that's possible, but maybe I'm missing something.

Reply:
Ok, discussed this with Alano and added examples for both invalid "flows".